src/bluesky/run_engine.py

```diff
@@ -2823,7 +2824,7 @@ def ignore_exceptions(self, val):
 
 def _set_span_msg_attributes(span, msg):
     span.set_attribute("msg.command", msg.command)
-    span.set_attribute("msg.args", msg.args)
+    span.set_attribute("msg.args", np.array(msg.args).tolist())
```

Comment:
Can we use `bluesky.utils.sanitize_np` instead?

Reply:
I didn't know about that, thanks!